man/systemd.service.xml

```diff
@@ -249,27 +268,25 @@
             anyway.</para></listitem>
           </itemizedlist>
 
-          <para>It is generally recommended to use <varname>Type=</varname><option>simple</option> for
+          <para>It is generally recommended to use <varname>Type=</varname><option>exec</option> for
           long-running services whenever possible, as it is the simplest and fastest option. However, as this
```

Comment:
uh, this is simply wrong. it's definitely *not* the "fastest" nor "simplest" option.